app/models/concerns/user_otp_methods.rb

```diff
@@ -15,7 +15,7 @@ def create_direct_otp
   end
 
   def generate_totp_secret
-    ROTP::Base32.random(20)
+    ROTP::Base32.random(20) # 160-bit secret, per RFC 4226
```

Comment:
not technically related but it was a comment I had before https://github.com/18f/identity-idp/pull/5604#discussion_r748431221